src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,39 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  user: User;
+  todo: Todo | null;
+  setIsOpenTodo: (b: boolean) => void;
+  setTodo: (v: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  user,
+  todo,
+  setIsOpenTodo = () => { },
+  setTodo = () => { },
+}) => {
+  const [loader, setLoader] = useState(true);
```

Comment:
Rename all boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isLoading, setIsLoading] = useState(true);
```